torchaudio/csrc/ffmpeg/stream_reader/packet_buffer.h

```diff
@@ -0,0 +1,16 @@
+#pragma once
+#include <torchaudio/csrc/ffmpeg/ffmpeg.h>
+
+namespace torchaudio {
+namespace io {
+class PacketBuffer {
+ public:
+  void push_packet(AVPacket* packet);
+  c10::optional<std::vector<AVPacketPtr>> pop_packets();
+  bool is_ready();
```

Comment:
Can we call this method something like `has_value`?
Because IMO, readiness is not well-defined for packet buffer, because packet buffer is more of a temp location where packets are just put.